wkcuber/cubing.py

```diff
@@ -173,9 +193,41 @@ def cubing_job(args):
                 raise exc
 
 
+def downsample_unpadded_data(buffer, target_mag, interpolation_mode):
```

Comment:
Nice :+1: I'd suggest to move this inside the downsampling.py module. Afterwards, feel free to merge :)